internal/runtimehandlerhooks/utils_linux.go

```diff
@@ -228,3 +243,44 @@ func fileExists(filename string) bool {
 
 	return !info.IsDir()
 }
+
+// calculateCPUSizeFromMask return the number of total cpus
+// given a CPU mask
+// this is useful for finding the complement of a mask.
+func calculateCPUSizeFromMask(mask string) int {
```

Comment:
How will this behave in case the highest Id cpu is currently masked?

Reply:
AFAIR, the full mask will be always given even if the highest IDs are masked (are zeros)

Reply:
```
sh-5.1# cat /proc/irq/default_smp_affinity
ff,ffffffff,ffffffff,ffffffff

sh-5.1# echo ff0 > /proc/irq/default_smp_affinity
sh-5.1# cat /proc/irq/default_smp_affinity
00,00000000,00000000,00000ff0
```

Reply:
Ok, I missed that indeed. If we assume the input is the full mask including zeros then this works fine.